test/generators/test_random_cell_complexes - Copy.py

```diff
@@ -0,0 +1,22 @@
+"""Test the random cell complex generators."""
+from toponetx.generators.random_cell_complexes import cc_np_complex
+
+
+class TestCcNpComplex:
+    """Test the `cc_np_complex` function."""
+
+    def test_zero_nodes(self):
+        """Test `cc_np_complex` for zero nodes."""
+        cc = cc_np_complex(0, 0.1)
+        assert cc.shape == (0, 0, 0)
+
+    def test_non_zero_nodes(self):
+        """Test `cc_np_complex` for nonzero nodes."""
+        cc = cc_np_complex(5, 0.1)
+        assert cc.shape[0] == 5
```

Comment:
Ideally a test case for initialization without the second argument (seed) must be added.